packages/blade/src/components/Card/CardRoot.web.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import React from 'react';
+import type { CardRootProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { castWebType, makeMotionTime } from '~utils';
+import { makeAccessible } from '~utils/makeAccessible';
+import { useIsMobile } from '~utils/useIsMobile';
+
+const StyledCardRoot = styled(BaseBox)<CardRootProps & { isPressed: boolean; isMobile: boolean }>(
+  ({ theme, isSelected, isFocused, scaleOnHover, isPressed, isMobile }) => {
+    const selectedColor = isSelected ? theme.colors.brand.primary[500] : 'transparent';
+    const selectedRing = `0px 0px 0px ${theme.border.width.thick}px ${selectedColor}`;
+    //  focused state
+    const focusRing = isFocused ? `, 0px 0px 0px 4px ${theme.colors.brand.primary[400]}` : '';
+
+    return {
+      // Selected state
+      // TODO: use thicker
+      boxShadow: `${selectedRing}${focusRing}`,
+      transitionDuration: castWebType(makeMotionTime(theme.motion.duration.xquick)),
+      transitionTimingFunction: castWebType(theme.motion.easing.standard.effective),
+      transitionProperty: 'transform, box-shadow',
+
+      // pressed state for mobile only
+      ...(isMobile &&
+        isPressed && {
+          transform: 'scale(0.95)',
+        }),
+
+      // Hover state for desktop only
+      ...(!isMobile &&
+        scaleOnHover && {
+          '&:hover': {
+            transform: 'scale(1.05)',
+          },
+        }),
+
```

Comment:
why do we have CardRoot then again Overlay? Shouldn't overlay handle all of these things?

Reply:
For react native, react native cannot use LinkOverlay trick. 

Plus we need attach onMouseOver, isFocused, isSelected all of the visual enhancements on the CardRoot itself, can't be done with LinkOverlay.